extension/src/popup/components/account/AssetDetail/styles.scss

```diff
@@ -3,6 +3,12 @@
     position: relative;
   }
 
+  &__list {
+    display: flex;
+    flex-direction: column;
+    gap: 1.5rem;
```

Comment:
NIT: would you mind using `pxToRem` here? It'd be great to start using that for all new styles in the repo. I'm hoping that helps us make sure our styles stay true to the Figma design by removing the extra step of converting to rem

Reply:
will do, thanks for reviewing!

Reply:
https://github.com/stellar/freighter/pull/1785/commits/646517f8b5946e80dc6cead47510bf7a5438d869